README.md

```diff
@@ -128,10 +128,18 @@ Quick start
 
 To see Great Expectations in action on your own data:
 
+You can install it using pip
 ```
     pip install great_expectations
-    great_expectations init
+```
+or conda
+```
+conda install -c conda-forge great-expectations
+```
+and then run
```

Comment:
<img width="585" alt="Screen Shot 2021-08-09 at 10 33 18 AM" src="https://user-images.githubusercontent.com/49923762/128723734-9260f9dd-7ce3-4e3a-ae25-4b3f686ec7ab.png">
Could you please left align all of these commands?

Reply:
done